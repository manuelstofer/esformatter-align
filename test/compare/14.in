var y = { blu: 1 };
var x = {
  ...x,
  bla: 5
};
