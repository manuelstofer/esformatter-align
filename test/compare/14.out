var x = {
       ...x,
  bla: 5
};
